src/DataAccess/Entities/Apprenticeship/Episode.cs

```diff
@@ -0,0 +1,28 @@
+﻿using SFA.DAS.Apprenticeships.Enums;
+
+namespace SFA.DAS.Apprenticeships.DataAccess.Entities.Apprenticeship;
+
+[Table("dbo.Episode")]
+[System.ComponentModel.DataAnnotations.Schema.Table("Episode")]
+public class Episode
+{
+    public Episode()
+    {
+        Prices = new List<EpisodePrice>();
+    }
+    
```

Comment:
Funky indentation

Reply:
It's a mixture of tabs and spaces I think which causes it to be funky or non funky depending on which viewer you are using and if it decides a tab is the same width as however many spaces it's used.